src/content/docs/Company/offboarding.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Offboarding
+description: Shorebird Offboarding
+template: doc
+sidebar:
+  order: 98
+---
+
+This document ensures a secure and consistent process when an employee exits the
+company, including revocation of access, return of company assets, and knowledge
+transfer. This is really the implementation details of the
+[Offboarding and Access Termination section of our Human Resource Security Policy](https://handbook.shorebird.dev/compliance/hr-security-policy/#offboarding-and-access-termination).
+
+Please note that this is a checklist and that not all items will apply to all
+employees. Managers should work with their employees throughout this process to
+ensure all required items are completed.
+
+## Exit Event
+
+**This starts when an employee has given notice or is terminated.**
+
+- Confirm resignation or termination in writing.
+- Determine last working day and communicate to relevant internal teams.
+- Schedule off boarding meetings with relevant people. As a starting point
+  please consider the following:
+  - Your Manager
+  - People Operations
+- Plan knowledge transfer sessions and identify documentation needs.
+- Set final day for employment in Rippling.
+
+## Company Property Recovery
+
+**Timeline for this to be created after an exit event. May be completed after
+final day given shipping times.**
+
+- Retrieve or confirm destruction of:
+  - Company-issued laptops, phones, or hardware tokens
```

Comment:
If this is trying to be a document for us, rather than for the auditors, then I'd suggest things like a shipping service here.  e.g. https://helloretriever.com/ worked fine in the past.